browser/src/canvas/CanvasSectionContainer.ts

```diff
@@ -159,6 +159,10 @@ enum DirtyType {
 	TileRange
 }
 
+interface Window {
```

Comment:
I don't like this. We will have some global definition for it. Could you just cast to `any` `window` where it is used?